internal/server.go

```diff
@@ -208,7 +208,7 @@ func (s *Server) LogoutHandler() http.HandlerFunc {
 		logger.Info("Logged out user")
 
 		if config.LogoutRedirect != "" {
-			http.Redirect(w, r, config.LogoutRedirect, http.StatusTemporaryRedirect)
+			http.Redirect(w, r, config.LogoutRedirect + "https://" + r.Host, http.StatusTemporaryRedirect)
```

Comment:
What is the value of r.Host? Will that be based on the URL used on the browser address bar?

Reply:
its the value of the host request header